crates/core/src/presentation.rs

```diff
@@ -0,0 +1,215 @@
+//! Verifiable presentation.
+
+use std::fmt;
+
+use crate::{
+    attestation::{Attestation, AttestationError, AttestationProof},
+    connection::{ConnectionInfo, ServerIdentityProof, ServerIdentityProofError, ServerName},
+    transcript::{PartialTranscript, TranscriptProof, TranscriptProofError},
+    CryptoProvider,
+};
+
+/// A verifiable presentation.
+#[derive(Debug)]
+pub struct Presentation {
+    attestation: AttestationProof,
+    identity: Option<ServerIdentityProof>,
+    transcript: Option<TranscriptProof>,
+}
```

Comment:
this feels like the right place for a 'quote' from the hardware, when available. todo for me  @maceip 

```diff
+   measurement: Option<HardwareReport>
```